src/citra_qt/updater/updater.cpp

```diff
@@ -110,9 +111,22 @@ QString UpdaterPrivate::ToSystemExe(QString base_path) {
 #endif
 }
 
+QFileInfo UpdaterPrivate::GetMaintenanceTool() const {
+#if defined(Q_OS_UNIX) && !defined(Q_OS_OSX)
+    const auto appimage_path = QProcessEnvironment::systemEnvironment()
+                                   .value(QStringLiteral("APPIMAGE"), QStringLiteral(""))
```

Comment:
```suggestion
                                   .value(QStringLiteral("APPIMAGE"), QStringLitera{})
```
just `{}` might be possible as well